internal/store/pod.go

```diff
@@ -1034,6 +1034,50 @@ var (
 					}
 				}
 
+				return &metric.Family{
+					Metrics: ms,
+				}
+			}),
+		},
+		{
+			Name: "kube_pod_overhead_mcpu_cores",
+			Type: metric.Gauge,
+			Help: "The pod overhead in mCPU cores for running a pod.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if p.Spec.Overhead != nil {
+					if cpu, ok := p.Spec.Overhead[v1.ResourceCPU]; ok {
+						ms = append(ms, &metric.Metric{
+							LabelKeys:   []string{"pod", "node"},
+							LabelValues: []string{p.Name, p.Spec.NodeName},
+							Value:       cpu.MilliValue(),
+						})
+					}
+
+					return &metric.Family{
+						Metrics: ms,
+					}
+				}
+			}),
+		},
+		{
+			Name: "kube_pod_overhead_memory_bytes",
+			Type: metric.Gauge,
+			Help: "The pod overead in bytes of memory for running a pod.",
+			GenerateFunc: wrapPodFunc(func(p *v1.Pod) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if p.Spec.Overhead != nil {
+					if mem, ok := p.Spec.Overhead[v1.ResourceMemory]; ok {
+						ms = append(ms, &metric.Metric{
+							LabelKeys:   []string{"container", "node"},
```

Comment:
I think you want to use the "pod" label here.

Reply:
Yes, yikes -- didn't commit the latest changes before pushing.  Thanks.